e2e/scripts/embed-and-install.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+# This script is used to test the embed capability of helmvm. It will pull the memcached helm chart and
+# embed it into the binary, issuing then a single node install. This script waits for the memcached pod
+# to be ready.
+
```

Comment:
i find it typically safer to set the following when writing scripts although pipefail can be difficult if you dont build with it in mind. take it or leave the suggestion

```suggestion

set -euo pipefail

```

Reply:
I was using it but then it was making things sorta flake because [this](https://github.com/replicatedhq/helmbin/pull/31/files#diff-e1c5d3a0f14947426c8657d3a54466390a0c27fef9f8d1e0474d63d175fae518R7) was failing from time to time. I mean, it is OK if it fails, it should keep trying.

In other words: how can I make the following command not to make the whole script fail with `set -euo pipefail` ?

```
ready=$(kubectl get nodes | grep -v NotReady | grep -c Ready)
```

Reply:
For extra context: when deploying with the token based authentication sometimes the `kubectl get nodes` was returning an error like "leader changed" or something. With the `-euo pipefail` the whole test was failing instead of just set "ready" to 0.

Reply:
perhaps something like

```
ready=$(kubectl get nodes | grep -v NotReady | { grep  -c Ready || true; }
```

Reply:
perhaps at the very least you can add `set -eu`

Reply:
Seems like the following did the trick:

```
ready=$(kubectl get nodes | grep -v NotReady | grep -c Ready || true)
```

Reply:
Resolving.